sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -496,19 +509,42 @@ case class Overlay(input: Expression, replace: Expression, pos: Expression, len:
     this(str, replace, pos, Literal.create(-1, IntegerType))
```

Comment:
Can we add `examples`? You could use `bin` function.

Reply:
> Can we add `examples`? You could use `bin` function.

Sorry, what is `bin` function? I don't know it.

Reply:
`bin` function in SparkSQL:

```
spark-sql> SELECT bin(13);
1101
```

Reply:
OK. I know you means. But there are exists some examples. I don't know the reason to use `bin` function. Thanks.

Reply:
I meant to add an example for this function under 502 lines. Since SparkSQL doesn't support binary literal you can use `bin` function in the example.

Reply:
or you can use other expressions like encode or unbase64 in the example to add.

Reply:
I think `bin` function used to translate long to binary but byte array.
I will look up other function so that supplement the examples.